src/app/common/services/swapy-protocol.service.ts

```diff
@@ -68,12 +85,12 @@ export class SwapyProtocolService {
     return build.networks[networkId].address;
   }
 
-  private async getEthPrice() {
+  public async getEthPrice() {
     return new Promise((resolve, reject) => {
       this.http.get(this.ethPriceProvider).subscribe(data => {
         resolve((data as any).bid);
       }, error => {
-        resolve(440.0);
+        reject(840);
```

Comment:
I miss this price :laughing: but I think we should throw an error instead

Reply:
You are right, I forgot to change this one.